packages/office-addin-debugging/src/port.ts

```diff
@@ -54,7 +54,11 @@ export function getProcessIdsForPort(port: number): Promise<number[]> {
   return new Promise((resolve, reject) => {
     const isWin32 = process.platform === "win32";
     const isLinux = process.platform === "linux";
-    const command = isWin32 ? `netstat -ano` : isLinux ? `netstat -tlpna | grep :${port}` : `lsof -n -i:${port}`;
+    const command = isWin32
+      ? `netstat -ano | findstr :${port}`
```

Comment:
do we need this `findstr` statement?

Reply:
This brings Win32 in line with the other platforms which filter the list down before looping over all the port bindings.  It's just an optimization and not critical.